### PR TITLE
Fix inconsistencies

### PR DESCRIPTION
**Pull request by kirsten-stallings**
Added section on fixing inconsistencies in the data

### CODE REVIEW

File: machine_learning_tutorial.ipynb [hunk 1]
```diff
@@ -962,21 +949,663 @@
...
+    "alphabetical_characters= sorted(list(set(str(name) for name in data[\"Name\"])))\n",
+    "print(len(alphabetical_characters))\n",
+    "alphabetical_characters[:14]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are at least 4 wats that `alliser thorne` is referenced but that is the same person! Not good if our model is trying to predict who said a particular line. We don't want the model deciding between `alliser thorn` and `alliser thorne`! Therefore, we need to fix these so there is one unique name for each character.\n",
```
Comment (jambran) [inline machine_learning_tutorial.ipynb]: what is "wat"?

File: tutorial_utils.py [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+
+
+class NamesToReplace():
+    """ 
+    this holds a list of names and their replacement names. 
+    The names are listed in the pairs(old_name, new_name)
+    To loop through this list, use the code:
+        for old_name, new_name in <object of type NamesToReplace>.names:
+"""
```
Comment (jambran) [inline tutorial_utils.py]: your end comment got off a tab :/
Some other pep8 issues in this file too

File: tutorial_utils.py [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    """
+    this function removes special characters from the list of authors and
+    returns a list of authors such that multiple authors are separated with a
+    '&'
+    param:: a set of writers
+    returns:: a set of all writers with special characters and 
+              extra characters removed
+    """
+    for old_symbol, new_symbol in [("\xa0"," "),("\u200a:"," "),
+    ("and","&"),("Story by ","")]:
```
Comment (jambran) [inline tutorial_utils.py]: the way this breaks over the line is difficult to read

File: machine_learning_tutorial.ipynb (hunk 1 above)
Comment (kirsten-stallings) [reply machine_learning_tutorial.ipynb]: ways. All the typos. I'll try to be better. Once i get all the info in there, i'll look over again.

File: tutorial_utils.py (hunk 2 above)
Comment (kirsten-stallings) [reply tutorial_utils.py]: i did that on purpose since it was sample code, but if it bothers you, i can put it back

Comment (kirsten-stallings) [reply tutorial_utils.py]: I went ahead and moved it over.

**Pull request merged**